cleaners/vlc.xml

```diff
@@ -19,20 +17,68 @@
 
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+    @app VLC media player
+    @url https://www.videolan.org/vlc/index.html
+    @os Windows, Linux, FreeBSD, NetBSD, OpenBSD
+    @cleanerversion v2.0.0
+    @cleanerdate 2019-03-18.2
+    @cleanerby Andrew Ziem (2010-09-23 - 2018-10-15) & tiemay (2014-06-06) & theatre-x (2015-02-05) & ROCKNROLLKID (2017-01-14) & Vincent DUVERNET - Nolmë Informatique (???) & Tobias B. Besemer (2019-03-18)
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-10 (... with an old version of the cleaner!)
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
+
 -->
 <cleaner id="vlc">
   <label>VLC media player</label>
   <description>Media player</description>
+  <!-- Don't know if the check of the task is really necessary... Tobias. -->
   <running type="exe" os="windows">vlc.exe</running>
   <running type="exe" os="linux">vlc</running>
+  <!--
+  BSD tasks not yet figured out!
+  <running type="exe" os="FreeBSD">vlc</running>
+  <running type="exe" os="NetBSD">vlc</running>
+  <running type="exe" os="OpenBSD">vlc</running>
+  Guess it would be:
+  <running type="exe" os="unix">vlc</running>
```

Comment:
I would not include the comments about BSD because they aren't confirmed. It's not informative.

They do seem like good guesses, so it would be OK to change line 38 from linux to unix

Reply:
They comments were for the next editor.

Does "Unix" include Linux?

Reply:
Btw.: I confirmed that the builds for this platforms exist...

Reply:
`os="unix"` includes Linux, the three BSD systems, and Darwin.

Reply:
I will add this to the docu and change it.

Reply:
I have added this now to the docu, too!

Reply:
Resolved in b459e5f21afed8b62e00d941db39d12b9abc2334